peer.js

```diff
@@ -253,6 +254,8 @@ TChannelPeer.prototype.setScoreStrategy = function setScoreStrategy(ScoreStrateg
 TChannelPeer.prototype.invalidateScore = function invalidateScore(reason) {
     var self = this;
 
+    self._weightedRange = null;
```

Comment:
Also I'm pretty surprised that we had to add this field at all to the peers, the original idea was that the heapElements _are_ the cache.  I was thinking that we'd modify the heap relation/comparator to work on ranges, then the code below would be modified to rescore by calling something like `self.scoreStrategy.getRange()` rather than `.getScore()` and passing that in to each of the heapElements.